src/web/routes/application/are-you-pregnant/validate.js

```diff
@@ -1,9 +1,49 @@
 const { check } = require('express-validator/check')
+const { toDateString } = require('../common/formatters')
+const { isValidDate, isDateMoreThanOneMonthAgo, isDateMoreThanEightMonthsInTheFuture } = require('../common/validators')
+const { YES, NO } = require('./constants')
+
+const addDateToBody = (req, res, next) => {
+  if (req.body.areYouPregnant === NO) {
```

Comment:
Can we have a test case for this scenario please. I'd also rename this method and extract the part that adds the Date to the body.

Reply:
Added unit tests but don't agree with "extract the part that adds the Date to the body" as this is the main purpose of the function.